blockchain/genesis_test.go

```diff
@@ -169,3 +329,38 @@ func TestSetupGenesis(t *testing.T) {
 		}
 	}
 }
+
+func genCypressGenesisBlock() *Genesis {
```

Comment:
Why you created another method for creating cypress genesis block instead of using `DefaultGenesisBlock` itself?

Reply:
I don't want to change `DefaultGenesisBlock`, so I made functions to generate genesis blocks of public networks. This method can be used in `genesis_test.go` only.